systems/WikibaseClient/02-Architecture_Constraints.md

```diff
@@ -3,3 +3,7 @@
 ## MediaWiki Integration
 
 WikibaseClient makes the [Repo's](../../Glossary.md#wikibase-repository) Entity data available to MediaWiki applications such as Wikipedia. Thus, it must integrate with and extend the functionality of MediaWiki, which is best achieved through a [MediaWiki extension](../../Glossary.md#mediawiki-extension) or a [collection thereof](../../Glossary.md#wikibase-extension).
+
+## Embedding [Entity](../../Glossary.md#entity) data in Client articles must be triggered through [Wikitext](../../Glossary.md#wikitext)
```

Comment:
Maybe it's just me, but I would prefer the Glossary links in the text, not in the section header.

Reply:
Yes, me too, now that you mention it :D